src/propterm.js

```diff
@@ -675,6 +675,22 @@ class PropTerm {
     getText() {
         return this.buffer.textArray.join('\r');
     }
```

Comment:
Adding these to the PropTerm class so that element containing the terminal doesn't have to be directly referenced/hardcoded.